docs/it/metadata_oidc_op.rst

```diff
@@ -129,10 +129,10 @@ L'EC di un OP DEVE configurare un metadata di tipo **"openid_provider"** DEVE co
     - Vedi `OpenID.Discovery#OP_Metadata`_. Vedi signature :ref:`supported_algs`.
     - |spid-icon| |cieid-icon|
   * - **request_object_encryption_alg_values_supported**
-    - Vedi `OpenID.Discovery#OP_Metadata`_. Vedi key encryption :ref:`supported_algs`.
+    - Fino a diversa indicazione di AgID, non deve essere incluso.
```

Comment:
questo testo non è neutrale per CIE

inoltre perché NON DEVE?
Se il RP supportasse la criptazione gli OP non devono supportarla?

Reply:
Gli OP SPID al momento non supporteranno questa cifratura, quindi gli RP non possono chiederla.

Reply:
LLGG SPID OIDC supporta questo parametro

Reply:
Vedi in https://github.com/italia/spid-cie-oidc-docs/pull/144#discussion_r1058153905 per request_object_encryption_enc_values_supported